lineman/app/components/mixins.scss

```diff
@@ -114,7 +114,7 @@ $navbar-height: 45px;
 #turn-off-angular {
   z-index: 100;
   position: fixed;
-  right: 10px;
+  left: 10px;
```

Comment:
I just bumped the turn off angular link to the other side so it doesn't block the start button